influx-sink.js

```diff
@@ -16,37 +23,102 @@ var defaults = {
   }
 }
 
-
-module.exports = function (opts) {
+module.exports = function (options) {
   var seneca = this
   var extend = seneca.util.deepextend
 
-  opts = extend(defaults, opts)
-  seneca.add({role: opts.role, hook: 'sink'}, handle_storage)
+  locals = extend(locals, options)
+
+  locals.log = bole(locals.plugin)
+  locals.client = influx(locals.influx)
+
+  locals.batch.list = {}
+  locals.batch.stream = through.obj(on_write)
```

Comment:
Ah but then I loose the functionality of write to batch size x or every y msecs.